includes/Core/Util/Input.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Input abstraction class.
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class Input
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Input {
+
+	/**
+	 * Gets a specific external variable by name and optionally filters it.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @link https://php.net/manual/en/function.filter-input.php
+	 * @param int    $type One of INPUT_GET, INPUT_POST, INPUT_COOKIE, INPUT_SERVER, or INPUT_ENV.
+	 * @param string $variable_name Name of a variable to get.
+	 * @param int    $filter [optional] The ID of the filter to apply. The manual page lists the available filters.
+	 * @param mixed  $options [optional] Associative array of options or bitwise disjunction of flags.
```

Comment:
The intentation here is should match our docs guidelines (parameter descriptions should all start at the same position).